src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -338,7 +338,7 @@ private void BuildFallbackBlock(TagHelperContent builder)
 
                 // Build the <meta /> tag that's used to test for the presence of the stylesheet
                 builder
-                    .AppendHtml("<meta name=\"x-stylesheet-fallback-test\" class=\"")
+                    .AppendHtml("<meta property=\"x-stylesheet-fallback-test\" content class=\"")
```

Comment:
Agree w/ @Eilon. Where HTML and tag helpers add content, the content should be XHTML-compliant. The minimized choice for this attribute goes against that. (And I'm not sure minimization is legal for anything except Boolean attributes.)

Separately where is the `property` attribute defined? Not mentioned in http://www.w3.org/TR/html5/document-metadata.html#the-meta-element